packages/tldraw/src/lib/ui/context/actions.tsx

```diff
@@ -1114,6 +1338,9 @@ export function ActionsProvider({ overrides, children }: ActionsProviderProps) {
 					editor.user.updateUserPreferences({ isSnapMode: !editor.user.getIsSnapMode() })
 				},
 				checkbox: true,
+				enabled() {
```

Comment:
seeing all the `return true` (and related to my comments about the `enabled` function implying true if missing), makes me think that maybe the function should be optional? on the other hand, it is good to force an author to think about the right value of `enable` should be when writing an action. probably it should be required, but just thinking out loud here.